### DISCUSSION TITLE
can't find fragment files for snare-see

### DISCUSSION THREAD

**Discussion by tatianub** [stuart-lab/signac#985] [Q&A] [answered]
Hi,
I am trying to reproduce results from the "joint RNA and ATAC analysis: SNARE-seq", but I can't access fragment file and fragment file index files, as the dropbox link is not functional anymore. Can you please provide new path for these files? Thanks a lot in advance!

**Comment by timoast** [comment, marked as answer]
These links are available in the vignettes on the develop branch: https://github.com/timoast/signac/blob/develop/vignettes/snareseq.Rmd

https://signac-objects.s3.amazonaws.com/snareseq/fragments.sort.bed.gz
https://signac-objects.s3.amazonaws.com/snareseq/fragments.sort.bed.gz.tbi

**Comment by tatianub** [comment]
thanks for your quick response!
